README.md

```diff
@@ -1,6 +1,14 @@
 # eap-microprofile-test-suite
  a small standalone test suite for MicroProfile on WF/EAP
 
+## Requirements for the testsuite
+ - JDK 8+
+ - Docker
+ 
+ Maven is not a hard requirement as `mvnw` is delivered with this repository.
```

Comment:
`Maven is not a hard requirement as ...` however maven is not mentioned among above requirements at all. It should be listed there as well.

Reply:
mvnw is in the repo, Maven is not really required

Reply:
Standalone Maven on the system ...

Reply:
If the main concern is that README shows mvn commands and not ./mvnw commands please create new PR / issue to fix this concern

Reply:
Yes, it's about recommended way how the test suite should be run. If using `mvn ...` command then Maven should be added as requirement.  If using `mvnw ...` command then Maven can be omitted.

Reply:
ok, here we go https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/55
Task for tomorrow :)